changelogs/current.yaml

```diff
@@ -20,6 +20,9 @@ minor_behavior_changes:
 - area: http2
   change: |
     Request authorities are now validated with a library function from QUICHE rather than nghttp2. This behavior change can be reverted by setting ``envoy.reloadable_features.http2_validate_authority_with_quiche`` to ``false``.
+- area: aws
+  change: |
+    Added support for fetching credentials from the AWS credentials file. The order of credential fetching methods is now: 1) environment variables, 2) the credentials file, 3) instance metadata, 4) task metadata.
```

Comment:
I think the order of 3 or 4 depends on whether the Envoy is running on Ec2 or ECS. So we can probably just mention that Environment is used before credentials file